lib/ngtcp2_conn.c

```diff
@@ -3816,10 +3816,10 @@ static int conn_recv_stream(ngtcp2_conn *conn, const ngtcp2_stream *fr) {
     if (fr_end_offset <= rx_offset) {
       return 0;
     }
+  }
 
-    if (strm->flags & NGTCP2_STRM_FLAG_STOP_SENDING) {
-      return 0;
-    }
+  if (strm->flags & NGTCP2_STRM_FLAG_STOP_SENDING) {
```

Comment:
This change ensures no data can be forwarded to application layer no matter whether the fin flag in the frame is set or not

Reply:
I think this particular issue has been fixed in draft-15 branch.

Reply:
It is better to move the code to common place ?

Reply:
In general yes.  For this particular change, the current code works fine and I don't feel any need for change.